src/components/ClientInfo/Name/NameOn.tsx

```diff
@@ -0,0 +1,177 @@
+                <div className="brain_btns">
+                  <div className="btn">
+                    <div className="btn_Title">Atlas</div>
+                    <div className="btnContainer">
+                      <div onClick={handleChangeBtn} className={activeBtnAtlas == "L" ? "btnActive" : "name_btn"}>L</div>
+                      <div onClick={handleChangeBtn} className={activeBtnAtlas == "X" ? "btnActive" : "name_btn"}>X</div>
+                      <div onClick={handleChangeBtn} className={activeBtnAtlas == "R" ? "btnActive" : "name_btn"}>R</div>
+                    </div>
+                  </div>
+                  <div className="btn">
+                    <div className="btn_Title">ShortLeg</div>
+                    <div className="btnContainer">
+                      <div onClick={handleChangeBtnShortLeg} className={activeBtnShortLeg == "L" ? "btnActive" : "name_btn"}>L</div>
+                      <div onClick={handleChangeBtnShortLeg} className={activeBtnShortLeg == "X" ? "btnActive" : "name_btn"}>X</div>
+                      <div onClick={handleChangeBtnShortLeg} className={activeBtnShortLeg == "R" ? "btnActive" : "name_btn"}>R</div>
+                    </div>
+                  </div>
+                  <div className="btn containerRogueMode">
+                    <div className="btn_Title titleRogueMode">Rogue Mode</div>
+                    <div className="btnContainer btnRogueMode">
+                      <div onClick={handleChangeBtnRogueMode} className={activeBtnRogueMode == "on" ? "btnActive" : "name_btn"}>on</div>
+                      <div onClick={handleChangeBtnRogueMode} className={activeBtnRogueMode == "off" ? "btnActive" : "name_btn"}>off</div>
+                    </div>
+                  </div>
+                </div>
+
+                <div className="brain">
+                    <Brain/>
+                </div>
+
+                <div className="intakeInfoText_results">
+                  <div className="results">
+                    <div>63bpm</div>
+                    <div>HR</div>
+                  </div>
+                  <div className="results">
+                    <div>10</div>
+                    <div>Resp</div>
+                  </div>
+                  <div className="results">
+                    <div>98%</div>
+                    <div>SpO2</div>
+                  </div>
+                </div>
+              </div>
+              <div className="alphaContainer">
+                <div className="alphaContainer_text">
+                  Alpha
+                </div>
+                <div className="alphaContainer_letters">
+                  <div className="letter">R</div>
+                  <div className="letter">L</div>
+                </div>
+                <div className="alphaContainer_dashboard">
+                  <IntakeAlpha/>
+                </div>
+              </div>
+            </div>
+        </div>
+    </div>
+  )
+}
```

Comment:
so many commented code in this file))

Reply:
Thank you, Dana for your comments, I really need some comments yet) I'm deleting all the others now))